### DISCUSSION TITLE
How to Remove Theme from Project

### DISCUSSION THREAD

**Discussion by txwizard** [community/community#23624] [Other Feature Feedback, Questions, & Ideas]
<p>Yesterday, I applied the Cayman theme to two of my GitHub project documentation collections, and it seems to be incompatible with the documents generated by <strong>DocFX</strong> and stored therein. Although I see a way to change the theme, I see no way to disable it, so that the pages render <em>without</em> a theme.</p>
<p>Please advise.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>When you choose a theme, GitHub adds a line to your repository’s <code>_config.yml</code> like this:</p>
<pre><code>theme: jekyll-theme-cayman
</code></pre>
<p>You can delete that line (or the entire file if there is nothing else in your <code>_config.yml</code>) and that should remove the theme.</p>
<p>I hope that helps!</p>

**Reply by Devil-kushwaha** [reply]
It's not working. Is there any other way

**Comment by Pandit98himanshu** [comment]
<p>Can you please guide step by step, how to delete “_config.yml” file or remove th etheme from my repo. I’m new to github. please help me.</p>

**Comment by lee-dohm** [comment]
<p>There is <a href="https://help.github.com/articles/deleting-files/" rel="nofollow noopener">a help article on deleting files</a> that should give you what you need.</p>
<p>I hope that helps!</p>

**Comment by jools66** [comment]
<p>where is the _config.yml file located?</p>

**Comment by 0xt0pus** [comment]
<p>1- Open Your Repositories.<br>
2. Selec the repository where the default theme is installed<br>
3.You will have some files in this repositories, here you will find _config.yml<br>
4. click on _config.yml (this will open it)<br>
5. Now You will find the trash like button in rght upper side as shown in below image.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/4/84c0b2a4928acb268a42c739cdd67220a4eac1dc.png" data-download-href="/uploads/short-url/iWnSvXCO2VZ59TTb3BlvQiHqdiI.png?dl=1" title="help" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/55015567/181078309-7c31548b-fb83-4cb7-9632-8913fd12951e.png" alt="help" data-base62-sha1="iWnSvXCO2VZ59TTb3BlvQiHqdiI" width="690" height="136" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84c0b2a4928acb268a42c739cdd67220a4eac1dc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">help</span><span class="informations">766×151 2.08 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
6. click on that delete button and this will delete the default theme for you.<br>
7. Now if you want to show your sie with the page link directly then bring your index.html fie in root directory and yahooo this is done

**Comment by montein56** [comment]
<p>Here might be something of value too.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/96069501/181078312-52370846-cb08-4e5b-b3d5-09a9951846b5.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site" target="_blank" rel="noopener nofollow ugc">Unpublishing a GitHub Pages site - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
I had to resort to this too. Removed the theme ‘Jekyll’.</p>
